radish34/logger/dev.Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:11.15
```

Comment:
Is this based on alpine? If not, can you use the alpine based node image instead?

Reply:
AFAIK it is not based on alpine. I sticked with `node:11.15` for the logger image, since most of the services used `node:11.15` except of `deploy` and `messenger` services (which use 12ish and 13ish versions). Since the services such as zkp rely on node-gyp for dependencies switching to alpine may cause some issues and/or need some workaround.